shared/chat/conversations-list/index.desktop.js

```diff
@@ -177,8 +173,7 @@ class ConversationList extends PureComponent<void, Props, void> {
             type='uniform' />
         </div>
       </div>
-      {this.props.children}
```

Comment:
Note the conversation itself is no longer a child. That makes these two connected components more decoupled and reduces rendering